docs/source/entities-advanced.mdx

```diff
@@ -0,0 +1,535 @@
+
+<ExpansionPanel title="With Rover CLI composition">
+
+1. In the Billing subgraph's schema, define the `Bill` entity exactly as it's defined in the Payments subgraph:
+
+    ```graphql{2-5}
+    # Billing subgraph
+    type Bill @key(fields: "id") {
+      id: ID!
+      amount: Int!
+    }
+    ```
+
+3. In the Billing subgraph, define resolvers for each field of `Bill` that are currently resolved in the Payments subgraph. These resolvers should behave identically to the resolvers in the Payments subgraph.
+
+3. Deploy the updated Billing subgraph to your environment.
+
+    At this point, the Billing subgraph can successfully resolve `Bill` objects, but the gateway doesn't _know_ this yet because its supergraph schema hasn't been updated.
+
+4. In the Payments subgraph, remove the `Bill` entity and its associated resolvers (**do not deploy this change yet**):
+
+    <CodeColumns>
+
+    ```graphql
+    # Payments subgraph
+    type Payment {
+      # ...
+    }
+    ```
+
+    ```graphql
+    # Billing subgraph
+    type Bill @key(fields: "id") {
+      id: ID!
+      amount: Int!
+    }
+    ```
+
+    </CodeColumns>
+
+5. Compose an updated supergraph schema with your usual configuration using `rover supergraph compose`.
+    * This updated supergraph schema indicates that the Billing subgraph resolves `Bill` entities, and the Payments subgraph _doesn't_.
+
+6. Assuming CI completes successfully, deploy an updated version of your _gateway_ with the new supergraph schema.
+    * When this deployment completes, the gateway begins resolving `Bill` fields in the Billing subgraph _instead of_ the Payments subgraph.
+
+7. Deploy an updated version of your Payments subgraph without the `Bill` entity.
+    * At this point it's safe to remove this definition, because the gateway is never resolving `Bill` with the Payments subgraph.
+
+We're done! `Bill` is now resolved in a new subgraph, and it was resolvable during each step of the migration process.
+
+</ExpansionPanel>
+
+<ExpansionPanel title="With managed federation">
+
+1. In the Billing subgraph's schema, define the `Bill` entity exactly as it's defined in the Payments subgraph:
+
+    <CodeColumns>
+
+    ```graphql
+    # Payments subgraph
+    type Bill @key(fields: "id") {
+      id: ID!
+      amount: Int!
+    }
+
+    type Payment {
+      # ...
+    }
+    ```
+
+    ```graphql{2-5}
+    # Billing subgraph
+    type Bill @key(fields: "id") {
+      id: ID!
+      amount: Int!
+    }
+    ```
+
+    </CodeColumns>
+
+2. Publish the updated Billing subgraph schema to Apollo.
+
+    * Note that you're publishing a composition error, because multiple subgraphs define the same fields of `Bill` without marking them [`@shareable`](#using-shareable). This creates a failed [launch](https://www.apollographql.com/docs/studio/launches/) in Apollo Studio. That's okay! Apollo Uplink continues to serve the most recent _valid_ supergraph schema to your gateway.
```

Comment:
I'm really bothered by this paragraph. We really should move away from suggesting _any_ course of action that result in a composition error and failed launches in Studio. At least for GA, we should be able to have `@override(from:)` ready so it can be used here (but, @clenfest, this does remind me that we should consider allowing that on types as a shortcut of having it on all fields, same as we allow for `@shareable`, and this exactly for that use case).

Now, we may or may not have `@override(from:)` committed for preview so there is the question of what we put in the doc in the meantime. Personally, I'd rather we recommend marking the entity `@shareable` first in the `Payments` subgraph, add it at `@shareable` in `Billing` and remove the `@shareable` only later. *And* that we explicitly document that this is a temporary solution that will be cleaned up for GA. But if we really want to keep the "go and publish a composition error" for preview, lets *please* at least remove the "That's okay!" sentence and replace it with a sentence that acknowledge that it's a ugly hack that is going to be fixed.

Reply:
Updated to use `@shareable`!